src/components/AddOn/AddOn.test.tsx

```diff
@@ -2,21 +2,21 @@ import React from 'react';
 import { render } from '@testing-library/react';
 import { Provider } from 'react-redux';
 import { BrowserRouter } from 'react-router-dom';
-import { MockedProvider } from '@apollo/react-testing';
+import { MockedProvider, MockLink } from '@apollo/react-testing';
 import { I18nextProvider } from 'react-i18next';
 
 import { store } from 'state/store';
 import AddOn from './AddOn';
 import i18nForTest from 'utils/i18nForTest';
-
+const mocklink = new MockLink([], false, { showWarnings: false });
```

Comment:
Doesn't this `{ showWarnings: false }` section defeat the purpose of the PR?

Reply:
@palisadoes 

I have added showWarning to false in certain files. This is because, we are testing certain areas where the test cannot be provided with a mock. In that case an empty mock array is passed along with mocklink. This makes sure that no data is passed to the array.

showWarnings: false makes sure that false warning are not given out.

The reason I need to do this is because we are testing frontend components with Mocks, and we cannot test the complete application state with that.

We need e2e tests for testing these areas, and without those tests, we need to add the showWarnings to false in certain tests.

Reply:
1. How can we document this in the code so that someone doesn't do this in all cases? 
1. How can we document why the line is excluded in the others?

Reply:
@palisadoes 

I am finding a solution to that. Will be back with a resolution to this shortly.